datenguide_python/output_transformer.py

```diff
@@ -35,6 +37,7 @@ def build_and_merge_data(
             globals()["datatemp{}".format(i)] = json_normalize(
```

Comment:
I did not catch these earlier, but the use of global variables should be avoided as much as possible. In particular the purpose of classes is to internally store temporary variables. One much better strategy to avoid globals() would be to not use `@staticmethod`  such that the methods have access to the class instance by means of the `self` argument. Temporary variables can then be stored as part of the instance.
In terms of naming convention *temp* is never a good name for a variable, because in a sense all variables are temporary so no essential meaning is conveyed by the name. Try to find names that capture the purpose of the variable more clearly. This can, admittedly, be very hard.

I would also strongly encourage the use of f-strings as opposed to the alternatives as we agreed on python>=3.6, which guarantees their availability. That means using `f"datatemp{i}"` instead of `"datatemp{}".format(i)`. (And ideally not with *temp*).